chat-client/package.json

```diff
@@ -19,6 +19,7 @@
     },
     "dependencies": {
         "@aws/chat-client-ui-types": "0.x.x",
+        "@aws/language-server-runtimes": "0.x.x",
```

Comment:
Chat client should not depend on runtimes package (as also mentioned in another comment).